server.js

```diff
@@ -13,10 +16,47 @@ serve(async (req) => {
     return new Response("ウノ、ひより、ヤユヨ、やまじ");
   }
   if (req.method === "GET" && pathname === "/today") {
+    //テスト実行：http://localhost:8000/today
     const date = new Date();
     // date.getMonth()で帰ってくる月の数字は0~11月なので+1して1~12月になるようにする
     return new Response(`${date.getFullYear()}-${date.getMonth()+1}-${date.getDate()}`);
   }
+  
+  if (req.method === "GET" && pathname === "/temp") {
+    //テスト実行：http://localhost:8000/temp?latitude=32.5453&longitude=123.4567
+    //入力(緯度経度)を整理
+    const params = new URLSearchParams(req.url.substring(req.url.indexOf("?")))
+    const latitude = params.get("latitude")
+    const longitude = params.get("longitude")
+    console.log(params)
+    
+    //気象庁のデータ
+    if(true||lastTime_getWeather!=new Date().getHours()){// 1h経過したか監視
```

Comment:
```suggestion
    if(lastTime_getWeather!=new Date().getHours()){// 1h経過したか監視
```

true入れたままだったのはここかな？
サジェスト入れておきます